app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -0,0 +1,265 @@
+
+### Example
+This example of `Timeline` creates a `Circle` which moves 200px in x direction over the duration of 5 seconds:
+
+![TimelineExample](/assets/images/javafx/animation/timeline-example.gif)
+```java
+Circle circle = new Circle(50, 150, 10, Color.GREEN);
+
+KeyValue x = new KeyValue(circle.translateXProperty(), 200);
+KeyFrame frame = new KeyFrame(Duration.seconds(5), x);
+Timeline timeline = new Timeline(frame);
+
+timeline.play();
+```
+
+## Interpolator
+The [Interpolator](javafxdoc:Interpolator) abstract class defines the rate of change at which values change over time, influencing the smoothness of animations.
+
+It provides several built-in implementations:
+- [Interpolator.DISCRETE](javafxdoc:Interpolator.DISCRETE): The `DISCRETE` interpolator creates a **sudden** transition between values without any intermediate steps.
+- [Interpolator.LINEAR](javafxdoc:Interpolator.LINEAR): The `LINEAR` interpolator produces a **constant** rate of change between values over time.
+- [Interpolator.EASE_IN](javafxdoc:Interpolator.EASE_IN): The `EASE_IN` interpolator starts the animation slowly and accelerates as it progresses.
+- [Interpolator.EASE_OUT](javafxdoc:Interpolator.EASE_OUT): The `EASE_OUT` interpolator starts quickly and slows down as it progresses.
+- [Interpolator.EASE_BOTH](javafxdoc:Interpolator.EASE_BOTH): The `EASE_BOTH` interpolator starts slowly, accelerates in the middle and slows down towards the end. It combines the characteristics of `EASE_IN` and `EASE_OUT`.
+
+Additionally, there are two static factory methods for [SPLINE](javafxdoc:Interpolator.SPLINE) and [TANGENT](javafxdoc:Interpolator.TANGENT) interpolation.
+
+[//]: # (ToDo: sync gifs)
+Here is a visualization of the Interpolator using the example from [Timeline](#example):
+### Discrete
+![Discrete](/assets/images/javafx/animation/interpolator/discrete-example.gif)
+### Linear
+![Linear](/assets/images/javafx/animation/interpolator/linear-example.gif)
+### Ease In
+![EaseIn](/assets/images/javafx/animation/interpolator/ease-in-example.gif)
+### Ease Out
+![EaseOut](/assets/images/javafx/animation/interpolator/ease-out-example.gif)
+### Ease Both
+![EaseBoth](/assets/images/javafx/animation/interpolator/ease-both-example.gif)
+
+## AnimationTimer
+
+The [AnimationTimer](javafxdoc:AnimationTimer) abstract class provides the lowest level option to create an animation.
+The `handle(long now)` method gets called in each frame while it is active. The timestamp `now` is the time of the current frame in nanoseconds and will be the same for all `AnimationTimer`s called during that frame.
+Additionally, the `AnimationTimer` adds `start()` and `stop()` methods to handle the lifetime of the animation.
+
+**Note:** The `handle` method will be called in the **JavaFX Application Thread** and thus shouldn't do heavy computations.
+
+
+## Conclusion
+The `javafx.animation` package offers a simple framework for creating dynamic animations within JavaFX applications.
```

Comment:
This first phrase of the conclusion is exactly the same as the first one in your article. Enforcing a message is great, but would be great if you do that with a similar phrase. You could approach the conclusion with what the reader has learnt throughout the tutorial, instead of repeating the introduction. And...do not be afraid to share that this tutorial taught _this_ and _that_ 

Eg: _Additionally, you are provided with various common `Interpolator` implementations._
would become more relevant to the reader as _Additionally, you have learnt how to control the progression of an animation via several `Interpolator` examples._